src/NuGet.Core/NuGet.Configuration/Settings/Items/FileClientCertItem.cs

```diff
@@ -66,16 +73,16 @@ internal FileClientCertItem(XElement element, SettingsFile origin)
                                                                     Origin?.ConfigFilePath ?? "<Config file path>"));
             }
 
-            AddAttribute(ConfigurationConstants.PathAttribute, path);
+            AddAttribute(ConfigurationConstants.PathAttribute, path!);
 
             if (!string.IsNullOrWhiteSpace(password))
             {
-                AddAttribute(ConfigurationConstants.PasswordAttribute, password);
+                AddAttribute(ConfigurationConstants.PasswordAttribute, password!);
```

Comment:
I really wish the compiler was smarter here, is there a way for us to tell the compiler that `string.IsNullOrWhiteSpace()` already checked for `null`?

Reply:
Okay thanks for looking into it!